mkosi

```diff
@@ -2674,7 +2733,10 @@ def install_boot_loader(args: CommandLineArguments, root: str, loopdev: Optional
             install_boot_loader_clear(args, root, loopdev)
 
         if args.distribution == Distribution.photon:
-            install_boot_loader_photon(args, root, loopdev)
```

Comment:
Same here, we now pass around `root` instead of `workspace` so the above statement was the right one.

Reply:
Fixed, thanks for catching this.